app/assets/javascripts/client/inventories/productEntryModalCtrl.js

```diff
@@ -7,17 +7,18 @@
   ProductEntryModalCtrl.$inject = [
     '$scope',
     'ProductEntry',
-    'Enums'
+    'ConstantsService',
+    'CheckboxService'
   ];
 
-  function ProductEntryModalCtrl($scope, ProductEntry, Enums) {
+  function ProductEntryModalCtrl($scope, ProductEntry, ConstantsService, CheckboxService) {
     var vm = this;
 
     vm.closeModal = function() {
       $scope.$emit('close-product-entry-modal');
     };
 
-    vm.enums = Enums;
+    vm.constants = ConstantsService.constants;
```

Comment:
This is somewhat related to my comments in your PR yesterday. The way I imagined this would work (and the way I tested it) was through using `require` in the route which corresponds to wherever the modal invocation button is. Check out cc13e6a23e10055c64e032feddf187888ab2b721